### ISSUE TITLE
remove underscores "_" from "Add existing Shows" search strings

### ISSUE THREAD

**Issue by cheese1** [pymedusa/Medusa#7737]
**Is your feature request related to a problem? Please describe.**
I'm currently switching from sickbeard to medusa and therefore have a lot of preexisting shows.
If i navigate from "Add existing Shows" then the directory name has "\_" instead of spaces, as i dont like spaces in them. Now I have to change every "\_" to " " in the search form which gets preoccupied with the directory name, to get the correct results.

**Describe the solution you'd like**
have an option to replace "\_" with " " in search forms in "Add existing Shows".

**Describe alternatives you've considered**
rename all directories to have spaces instead of "\_". after adding i would need to revert that and enter every single show to save to the correct path. no real solution.

**Comment by below9k** [comment]
I've heard rumors the database is able to be migrated to medusa by just copying the `sickbeard.db` file from sickchill/rage to the medusa install.
This didn't work for me, but maybe it could for you. I ended up adding all 1k+ of my shows I have manually.

side note - if you go the manual adding route: be careful importing a bunch of shows at once. I had issues where some shows would *always* hang and medusa wouldn't proceed. I ended up having to bounce the service and re-add all the episodes in the queue
